### ISSUE TITLE
[FEATURE] PTZ controls

### ISSUE THREAD

**Issue by jamitupya** [roflcoopter/viseron#449]
I have a bunch of cameras with PTZ, it would be great if I could utilise this. 

I also would like to be able to set zones inside this field of view that takes into consideration directionality and zoom levels.

This would help focus on safety with sensors and roving cameras.

**Comment by TychoWerner** [comment]
It would be awesome if the camera could move by itself and check out beyond it's normal vision and keep interesting things (based on what it sees) in shot

**Comment by roflcoopter** [comment]
Adding ptz through ONVIF should be doable.

Moving the camera to keep an object in frame will probably be very difficult since it would need a lot of calibration

**Comment by ulanking** [comment]
Isn't this partially available via the ONVIF PTZ  component using presets?

**Comment by roflcoopter** [comment]
Yes that is correct, however there is no UI for it yet sadly, will be implemented with #1205
